# Data FROM companion computor TO MAVLink stream

uint64 timestamp	# Local time since system start (microseconds)
bool input_on		# Switch on
bool input_off		# Switch off
